app/models/trainee.rb

```diff
@@ -194,6 +194,14 @@ class Trainee < ApplicationRecord
     )
   end)
 
+  scope :course_not_yet_started, -> { not_draft.where("itt_start_date > ?", Time.zone.now) }
```

Comment:
`course_not_yet_started ` and `in_training ` lifted from this PR: https://github.com/DFE-Digital/register-trainee-teachers/pull/2450 which includes tests.